### PR TITLE
docker-env command: Added support for multiple shells with no-proxy and unset flags

### LINKED ISSUE #28: Support for Windows
Minikube should eventually run on Windows.

### PR DESCRIPTION
**Pull request by jimmidyson**
Fixes #28

Thanks to @rodcloutier for this - I've just tested it out & tidied up a few minor things.

### CODE REVIEW

**Comment by jimmidyson** [comment]
@rodcloutier Would you be able to comment something like "@googlebot I'm okay with these commits being contributed to Google"? You've already stated that in https://github.com/kubernetes/minikube/issues/28#issuecomment-234691707 but googlebot won't be happy otherwise...

**Comment by rodcloutier** [comment]
@googlebot I am okay with these commits being contributed to Google

**Comment by jimmidyson** [comment]
@dlorenc Any idea how to appease the CLA bot?

**Comment by dlorenc** [comment]
It looks like the CLA bot won't mark this as "OK", but as long as we follow the rules from the comment we're fine.

@aaron-prindle, merge once you take a look.

File: cmd/minikube/cmd/env.go [hunk 1]
```diff
@@ -14,47 +14,251 @@ See the License for the specific language governing permissions and
...
+		shellCfg.Suffix = "\n"
+		shellCfg.Delimiter = ""
+	}
+
+	return shellCfg, nil
+}
+
+func executeTemplateStdout(shellCfg *ShellConfig) error {
+	t := template.New("envConfig")
+	tmpl, err := t.Parse(envTmpl)
```
Comment (dlorenc) [inline cmd/minikube/cmd/env.go]: You could simplify this with template.Must to skip the error handling..

**Comment by dlorenc** [comment]
Also this is currently failing the integration tests. They're still not blocking pre-submits, so it's not a huge deal, but the regex here needs to be changed to handle quotes:
https://github.com/kubernetes/minikube/blob/master/test/integration/util/util.go#L88

	re := regexp.MustCompile("export (.+?)=\"?(.+?)\"?\n")
works for me.

**Comment by dlorenc** [comment]
@minikube-bot retest this please

**Comment by aaron-prindle** [comment]
LGTM.  Holding off on merging after discussing with @dlorenc about failing integration tests.  Please let me know how to proceed, thanks.

**Comment by jimmidyson** [comment]
I'll fix up the integration tests & other comment today.

File: cmd/minikube/cmd/env.go (hunk 1 above)
Comment (jimmidyson) [inline cmd/minikube/cmd/env.go]: Fixed.

**Comment by jimmidyson** [comment]
Ready for review.

**Comment by dlorenc** [comment]
Nice. LGTM. I finally got the test link to appear correctly in "details", too!

**Pull request merged**